spec/JSON-RPC.md

```diff
@@ -2245,6 +2246,38 @@ No parameters
 
 [Back to **List of methods**](#list-of-methods)
 
+## net_recentNetworkUsage
+Gets the recent network usage.
```

Comment:
Gets the recent network usage and returns an object.

Reply:
I'll change the next sentence to 'The return type is an object.'